maa-cli/src/run/copilot.rs

```diff
@@ -0,0 +1,207 @@
+use std::{
+    fs::{self, File},
+    io::{self, BufReader, Write},
+};
+
+use super::{run, Result};
+use crate::{
+    config::task::{
+        default_variants, task_type::TaskType, value::input::Input, Strategy, Task, TaskList, Value,
+    },
+    dirs::Dirs,
+    installer::maa_core::find_resource,
+    object,
+};
+use anyhow::Context;
+use reqwest::blocking::Client;
+use serde_json::Value as JsonValue;
+
+pub fn copilot(dirs: &Dirs, link: String) -> bool {
```

Comment:
I think this function should return `Result<()>` instead of `bool` to indicate the status.